api/v1/sriovnetworkpoolconfig_types.go

```diff
@@ -21,6 +21,10 @@ type SriovNetworkPoolConfigSpec struct {
 	// Drain will respect Pod Disruption Budgets (PDBs) such as etcd quorum guards,
 	// even if maxUnavailable is greater than one.
 	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty"`
+
+	// +kubebuilder:validation:Enum=shared;exclusive
+	// RDMA subsystem. Allowed value "shared", "exclusive".
+	RdmaMode string `json:"rdmaMode,omitempty"`
```

Comment:
This option is only valid for systemd mode? 
Do we want to document this somehow?

Reply:
done as log message in a SriovNetworkPoolConfig controller

Reply:
@e0ne cant we set this using module parameter ? 

https://github.com/torvalds/linux/blob/55027e689933ba2e64f3d245fb1ff185b3e7fc81/drivers/infiniband/core/device.c#L128C1-L128C19